### ISSUE TITLE
Pom: Change snapshot check to daily instead of always

### ISSUE THREAD

**Issue by davidgraeff** [openhab/openhab2-addons#5416]
@openhab/2-x-add-ons-maintainers Would it be a problem to change the snapshot download rate to daily instead of always?

Locally I'm going crazy with the current default and I assume other non experienced developers run into the same issue. And I know of the `-DnoOhSnapRepo` command line flag, but I tend to forget that all the time and need to look it up again and Eclipse also doesn't use it and happily redownloads snapshot data all the time.

It also increases ci build times.

Is there actually a reason to have that set to "always"?
Because the openHAB core doesn't change that often and in a binding development environment core bundles will be provided by a distro maven reactor anyway.

**Comment by wborn** [comment]
On other projects I've set snapshot updates to `never` and use -U if I choose to update these and on build servers:

```
 -U,--update-snapshots                  Forces a check for missing
                                        releases and updated snapshots on
                                        remote repositories
```

But I can imagine all this causes issues for less savvy Maven users and they are better off with the current always configuration.

**Comment by davidgraeff** [comment]
I prefer "-U" as well. But daily should be good enough for those users, as they are probably not developing on the core as well or depending on less than 24h old core changes. It is a comprise that helps CI and local development (except the first time of the day of course).

**Comment by wborn** [comment]
There's also `-DnoOhSnapRepo` you can use nowadays. But IMHO it should set the update policy instead of removing the repo. It would cause issues if you don't have these artifacts in your local repo.

**Comment by davidgraeff** [comment]
@openhab/2-x-add-ons-maintainers Can please everyone vote with a yes / no or voice his objections? Thanks.

Tl;dr: Change snapshot downloading to "daily" instead of "always" (which is already super conservative, I would even suggest "never").

**Comment by J-N-K** [comment]
Daily would be ok for me.

**Comment by martinvw** [comment]
I’m okay with daily 👍

**Comment by cweitkamp** [comment]
:+1: for daily. No objection.

**Comment by kaikreuzer** [comment]
This means that we have to use `-U` in our build jobs and must not forget this, correct?

**Comment by martinvw** [comment]
> This means that we have to use -U in our build jobs and must not forget this, correct?

Especially for release and nightly maybe not for PRs I would like the speed improvement of that

**Comment by kaikreuzer** [comment]
Ok, let's experiment with that and see where we have issues.
